hack/generator/pkg/astmodel/errored_type.go

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
```

Comment:
You may want to do a test to ensure that this type makes it all the way through the pipeline. There are at least a few stages where wrapper types got dropped when I did this for `FlagType`.

See [here](https://github.com/matthchr/k8s-infra/blob/744a47fda37545c6e4ce6283e284f102dfc87c2c/hack/generator/pkg/codegen/pipeline_create_arm_types.go#L228) and [here](https://github.com/Azure/k8s-infra/pull/309/files#diff-ac2b16201a4dd7f4ffd0044fae998aba3391504a0a3d3d98d78ed08f2e5a1e1a) for example.

Reply:
I'll deal with these cases when I start using it 🙂